cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/CloudRegions.java

```diff
@@ -20,9 +22,27 @@
 
     public CloudRegions(Map<Region, List<AvailabilityZone>> cloudRegions, Map<Region, String> displayNames, Map<Region, Coordinate> coordinates,
             String defaultRegion, boolean regionsSupported) {
-        this.cloudRegions = cloudRegions;
-        this.displayNames = displayNames;
-        this.coordinates = coordinates;
+        this.cloudRegions = cloudRegions
+                .entrySet()
+                .stream()
+                .sorted(Comparator.comparing(o -> o.getKey().getRegionName()))
+                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue,
+                        (oldValue, newValue) -> oldValue, LinkedHashMap::new));
```

Comment:
You can make a function for this. As I see these lines are the same as for `dispalyNames` and `coordinates`